apps/nfid-frontend/src/integration/ft/impl/ft-impl.ts

```diff
@@ -42,6 +42,10 @@ export class FTImpl implements FT {
     return this
   }
 
+  isInited(): boolean {
+    return this.tokenBalance !== undefined
```

Comment:
let's add. boolean field `inited`

Reply:
it's already return boolean.
Didn't get you.
Did you mean return object { inited: boolean } ?
@olekid

Reply:
Oleksii means to change method isInited to boolean field inited on your class